tce/opset/base/implicit_lsu.cc

```diff
@@ -0,0 +1,300 @@
+/*
+    Copyright (c) 2002-2020 Tampere University.
```

Comment:
(c) belongs to you or Eindhoven University. We have only agreed on licensing, not copyright transfer.

Reply:
Fixed